contribs/gmf/apps/mobile/index.html

```diff
@@ -30,6 +30,10 @@
         ng-click="mainCtrl.toggleRightNavVisibility()">
       </button>
       <div class="overlay" ng-click="mainCtrl.hideNav()"></div>
+      <gmf-mobile-geolocation
+        gmf-mobile-geolocation-map="mainCtrl.map"
```

Comment:
You should be able to use one time binding here: `gmf-mobile-geolocation-map="::mainCtrl.map"` (untested)

The same should be possible for the options 
(`gmf-mobile-geolocation-options="::mainCtrl.mobileGeolocationOptions"`)